apps/ssr-test-app/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": ["dom", "dom.iterable", "esnext"],
+    "allowJs": true,
+    "skipLibCheck": true,
+    "strict": false,
+    "forceConsistentCasingInFileNames": true,
+    "noEmit": true,
+    "esModuleInterop": true,
+    "module": "esnext",
+    "moduleResolution": "node",
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "jsx": "preserve",
+    "incremental": true
```

Comment:
I'm guessing you probably just copied this from somewhere but could you please alphabetize these so it makes it easier for humans to find things in this list? Thanks!

Reply:
Yeah, the tsconfig file was auto-generated by nextjs. I'll go through and make the changes you mentioned.

Reply:
Reordered!

(Do you ever think about how alphabetical order is completely arbitrary and really has no meaning? Like who decided that this was going to be the order of the letters? It's not like there's any innate quality to each of the letters to order them by. Did we just collectively decide that was going to be the order and rolled with it, or did some governing body decide?)

Reply:
[https://en.wikipedia.org/wiki/Alphabetical_order#History](https://en.wikipedia.org/wiki/Alphabetical_order#History)